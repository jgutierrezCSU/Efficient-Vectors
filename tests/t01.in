printStat
5
5 2 7 4 3
printStat
10
72	34	41	85	97	61	55	27	3	60
printStat
10
15	34	41	85	60	61	55	27	72	97
