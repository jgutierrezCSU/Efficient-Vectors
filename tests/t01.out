Array: 
5 2 7 4 3 
Min, max, average:
2 7 4.2
Array: 
72 34 41 85 97 61 55 27 3 60 
Min, max, average:
3 97 53.5
Array: 
15 34 41 85 60 61 55 27 72 97 
Min, max, average:
15 97 54.7
